drivers/include/net/netdev/ieee802154.h

```diff
@@ -119,7 +119,7 @@ typedef struct {
     uint8_t page;                           /**< channel page */
     uint16_t flags;                         /**< flags as defined above */
     int16_t txpower;                        /**< tx power in dBm */
-#if IS_USED(MODULE_IEEE802154_SECURITY) || defined (Doxygen)
+#if IS_USED(MODULE_IEEE802154_SECURITY) || defined(DOXYGEN)
```

Comment:
here and below, it seems these commits are actually 2: one for fixing typos/misspelled stuff and the other for prefixing `IEEE802154_SEC`.

Reply:
I split it up in more commits